saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -1700,7 +1709,13 @@ def create_single_fulfillment(
                 )
                 return None
 
-            if order_line.line.variant.id != variant.id:
+            if (
+                variant
+                and order_line.line.variant
+                and order_line.line.variant.id != variant.id
+                or (variant and not order_line.line.variant)
+                or (not variant and order_line.line.variant)
+            ):
```

Comment:
Could we split this into smaller part. I'm thinking about introducing variables to dedicated check, like: `line_variant_mismatch` and `missing_line_variant` and then use it here. What do you think?

Reply:
Sure, it makes sense.